CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
+- Audience parameter throws `InvalidAudienceError` when application does not specify an audience, but the token does. [#336][336]
```

Comment:
Add `[336]: https://github.com/jpadilla/pyjwt/pull/336` towards the end of the file.